policy/modules/system/systemd.te

```diff
@@ -1144,6 +1144,67 @@ optional_policy(`
 	dbus_watch_system_bus_runtime_named_sockets(systemd_resolved_t)
 ')
 
+#########################################
+#
+# Socket-proxyd local policy
+#
+## <desc>
+##  <p>
+##  Allow systemd-socket-proxyd to bind any port instead of one labelled
+## with systemd_socket_proxyd_port_t.
+##  </p>
+## </desc>
+gen_tunable(systemd_socket_proxyd_bind_any, false)
+
+## <desc>
+## <p>
+## Allow systemd-socket-proxyd to connect to any port instead of
+## labelled ones.
+## </p>
+## </desc>
+gen_tunable(systemd_socket_proxyd_connect_any, false)
+
+type systemd_socket_proxyd_t;
+type systemd_socket_proxyd_exec_t;
+init_daemon_domain(systemd_socket_proxyd_t, systemd_socket_proxyd_exec_t)
+allow init_t systemd_socket_proxyd_t:process2 { nnp_transition nosuid_transition };
```

Comment:
My reference policy-fu is not that great when it comes to interfaces.  My understanding that the `init_daemon_domain()` interface was supposed to define this transition as many other `init_*domain()` interfaces do, e.g. `init_domain()`, `init_script_domain()`,  and `init_spec_domain()`.  I would appreciate a pointer re: how to modify the rules in such a way that it would pass the linter and be accepted upstream.

Reply:
Ok, I looked closely and `init_daemon_domain()` includes a call to `init_domain()`, so that `allow` was unnecessary here.  Hopefully, I am done with adjustments and it will pass all the checks now (including the linting one).